packages/core/src/scss/components/checkout/ratepayForm/_ratepayForm.scss

```diff
@@ -0,0 +1,14 @@
+.dynamic-form-field--ratepayPhoneCountryCode {
+  display: inline-flex;
+  width: 20%;
+}
+
+.dynamic-form-field--ratepayPhoneNumber {
+  display: inline-flex;
+  width: calc(80% - 10px);
+  margin-left: 10px;
```

Comment:
Can we avoid using px values directly, we have utility methods like `remCalc` that help with responsiveness.

Reply:
fixed